scripts/xnat/sibis.py

```diff
@@ -7,4 +7,8 @@ def logging(id, message, **kwargs):
     log = dict(experiment_site_id=id,
                error=message)
     log.update(kwargs)
-    print(json.dumps(log, sort_keys=True))
+    try:
+        print(json.dumps(log, sort_keys=True))
+    except Exception as e:
+        print("ERROR: Failed to serialize to JSON: %s" % e)
+        print("Original log message: %s" % log)
```

Comment:
It be that experiment in check_phantom_scans:
146     except:
147         error = "ERROR: failed to get data for experiment"
148         sibis.logging(experiment, error)
149         return
is not serializable to JSON, in that case we could do:
sibis.logging(experiment.attrs, error)
But I couldn't reproduce the error to check it.